astronomer/providers/amazon/aws/sensors/batch.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Dict
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.sensors.batch import BatchSensor
+from airflow.utils.context import Context
+
+from astronomer.providers.amazon.aws.triggers.batch import BatchSensorTrigger
+
+
+class BatchSensorAsync(BatchSensor):
+    """
+    Asks for the JOB ID of the Batch Job execution asynchronously until it
+    reaches a failure state or success state.
```

Comment:
```suggestion
    Asks for the state of the Batch Job execution asynchronously  until it reaches a failure state or success state
```

Reply:
it needs job id, not the state of the batch job.

Reply:
done.